jquery-cheat-sheet.md

```diff
@@ -42,8 +44,50 @@ $(document).ready(function(){
   }
 });
 ```
+## Selecting elements
+
+
+## Modify content
+
+- `.html()`
+  - get or set the HTML contents
+    - get: no argument, know that it returns the innerHTML of the first jQuery object
+
+  ```javascript
+    $(".awesome").html()
+    // returns the innerHTML of the first element in the jQuery object
+  ```
+
+    - set: one argument that you want the html content to be
+
+  ```javascript
+    $(".awesome").html("this is awesome!")
+    // returns the innerHTML of the of all elements in the jQuery object to be "this is awesome!"
+  ```
+
+- `.text()`
+  - get or set the text contained, it will strip the HTML elements
+  - same as `.html()` only it returns the text of all jQuery objects selected for the getter portion
+
```

Comment:
suggest a simple practical example demonstrating the difference between `html()` and `text()`. (e.g. a div with a paragraph of text inside, demo that `html()` on the div will include the `<p>` tags in the string.